src/Microsoft.ML.AutoML/API/BinaryClassificationExperiment.cs

```diff
@@ -90,11 +90,6 @@ public enum BinaryClassificationMetric
     /// </summary>
     public enum BinaryClassificationTrainer
     {
-        /// <summary>
-        /// See <see cref="AveragedPerceptronTrainer"/>.
-        /// </summary>
-        AveragedPerceptron,
```

Comment:
>AveragedPerceptron [](http://example.com/codeflow?start=8&length=18)

Is there any app-copat concern removing public stuff from here?

Reply:
This API is still unstable so we can change it. But it will affect code that uses `AveragePerceptron`.

Reply:
Will be good if somehow document this breaking change.

Reply:
For this PR I add an api break change label to it so we can document it in the next release note. 
And let me talk to Eric and Luis on how to document it.